implementations/rust/ockam/ockam_api/src/cli_state/cli_state.rs

```diff
@@ -41,6 +41,32 @@ pub struct CliState {
     database: SqlxDatabase,
     application_database: SqlxDatabase,
     tracing_enabled: TracingEnabled,
+    tx: Option<tokio::sync::broadcast::Sender<ChannelMessageType>>,
+}
+
+const DEFAULT_CHANNEL_CAPACITY: usize = 16;
+pub type ChannelMessageType = String;
+pub const CHANNEL_POISON_PILL_MESSAGE: &str = "exit";
+pub const CHANNEL_MESSAGE_DISPLAY_DELAY_MS: u64 = 1000;
+
+// Broadcast channel support.
+impl CliState {
```

Comment:
Since this is a new concern for the `CliState` I'd rather see it in its own file.

Reply:
done